src/services/azureResourceService.ts

```diff
@@ -0,0 +1,62 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as mssql from 'vscode-mssql';
+import * as vscode from 'vscode';
+import { AccountStore } from '../azure/accountStore';
+import { AzureController } from '../azure/azureController';
+import { AzureResourceController } from '../azure/azureResourceController';
+import { Location } from '@azure/arm-subscriptions';
+import { ResourceGroup } from '@azure/arm-resources';
+import { Server } from '@azure/arm-sql';
+import providerSettings from '../azure/providerSettings';
+
+export class AzureResourceService implements mssql.IAzureResourceService {
+
+	private _accountStore: AccountStore;
+	constructor(
+		private _azureController: AzureController,
+		private _azureResourceController: AzureResourceController,
+		private _context: vscode.ExtensionContext) {
+		this._accountStore = new AccountStore(this._context);
```

Comment:
We shouldn't be making our own account store. We should be using the one from the azure account service.